### PR TITLE
Add with_scores argument to control using scores in zsets

### PR DESCRIPTION
**Pull request by smock**
@prog8 I ran into troubles with the boolean clause limit when using zsets as the scores are retrieved by default.  This p.r. adds a parameter to control the behavior.

### CODE REVIEW

**Comment by otisg** [comment]
Seems tests failed, @smock

File: src/main/java/com/sematext/solr/redis/command/ZRange.java [hunk 1]
```diff
@@ -14,9 +14,14 @@
     final String key = ParamUtil.assertGetStringByName(params, "key");
     final long start = ParamUtil.tryGetIntByName(params, "range_start", 0);
     final long end = ParamUtil.tryGetIntByName(params, "range_end", -1);
+	final boolean withScores = ParamUtil.tryGetBooleanByName(params, "with_scores", true);
```
Comment (lstrojny) [inline src/main/java/com/sematext/solr/redis/command/ZRange.java]: Spacing seems off

**Comment by smock** [comment]
@otisg tests are now passing

**Comment by otisg** [comment]
LGTM @prog8

**Comment by prog8** [comment]
Looks OK

**Comment by lstrojny** [comment]
@smock would you mind adding a bit of test coverage for the functionality?

**Pull request merged**